libs/application/templates/financial-aid/src/lib/utils.ts

```diff
@@ -1,8 +1,10 @@
 import { parsePhoneNumberFromString } from 'libphonenumber-js'
+import * as kennitala from 'kennitala'
 
 const emailRegex = /^[\w!#$%&'*+/=?`{|}~^-]+(?:\.[\w!#$%&'*+/=?`{|}~^-]+)*@(?:[A-Z0-9-]+\.)+[A-Z]{2,6}$/i
 export const isValidEmail = (value: string) => emailRegex.test(value)
 export const isValidPhone = (value: string) => {
   const phone = parsePhoneNumberFromString(value, 'IS')
   return Boolean(phone && phone.isValid())
 }
+export const isValidNationalId = (value: string) => kennitala.isValid(value)
```

Comment:
Didn't know about this package, we could now skip having this in utils and instead use the package everywhere where we validate national id.